backend/src/app.module.ts

```diff
@@ -0,0 +1,33 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+import { AuditLogModule } from './audit/auditLog.module';
+import { APP_INTERCEPTOR } from '@nestjs/core';
+
+import { AuthModule } from './auth/auth.module';
+import { PrismaModule } from '../prisma/prisma.module';
+import { PrismaService } from '../prisma/prisma.service';
+import { TokenExpiryInterceptor } from './auth/token-expiry.interceptor';
+import { TriageModule } from './triage/triage.module';
+import { NatsModule } from './nats/nats.module';
+
+@Module({
+  imports: [
+    ConfigModule.forRoot({
+      envFilePath: '.env',
+      isGlobal: true,
```

Comment:
I recommend adding environment variables validation to make sure all envs are defined during deployment.
```suggestion
      isGlobal: true,
      validate,
```

Reply:
I’ve added the validate option to ConfigModule.forRoot to ensure all required environment variables are validated during deployment.